examples/udp_bind_public.rs

```diff
@@ -1,4 +1,4 @@
-//! Demonstrates how bind to public address (using UPnP) and receive some data
+//! Demonstrates how to bind to public address (using `UPnP`) and receive some data
```

Comment:
Shame it demands "`" around UPnP. Wasn't there some way to disable this or add UPnP to its list of allowed words?

Reply:
There is: https://github.com/maidsafe/crust/blob/dev/clippy.toml
I just figured that if markdown was rendered, those ticks would actually make sense.
What do you think?

Reply:
How so? We don't have a type or trait in our code named `UPnP`.

Reply:
I'm just going to approve this though since it's not worth blocking this PR over something so trivial.